src/modules/fancyzones/editor/FancyZonesEditor/CanvasZone.xaml.cs

```diff
@@ -271,7 +271,7 @@ private void UpdateFromSnappyHelpers()
             Model.Zones[ZoneIndex] = rect;
         }
 
-        private static int zIndex = 0;
+        private static int zIndex;
```

Comment:
What is the reason to not explicitly set to zero?

Reply:
FxCop throws error.  It isn't needed to set when it is the default value.